senders/telegram/send.go

```diff
@@ -46,11 +46,18 @@ func (sender *Sender) SendEvents(events moira.NotificationEvents, contact moira.
 
 func (sender *Sender) buildMessage(events moira.NotificationEvents, trigger moira.TriggerData, throttled bool, maxChars int) string {
 	var buffer bytes.Buffer
+	var descr string
```

Comment:
I do not like the reduction of word description here. I do not see the strong reasons to use it instead of `description`

Reply:
Or it can be `desc`, since we use `trigger.Desc`.